remotespark/kernels/kernelmagics.py

```diff
@@ -143,6 +169,7 @@ def configure(self, line, cell="", local_ns=None):
         self.info("")
 
     @cell_magic
+    @_event
```

Comment:
Do we want Spark to have its own event? It seems to me Command events would cover it?